tests/sls/homu/toml.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+import toml
+
+from tests.util import Failure, Success
+
+
+def run():
+    config_path = os.path.join('/home', 'servo', 'homu', 'cfg.toml')
```

Comment:
It looks strange to see `os.path.join` when we have an explicit `/` in the arguments. Because I don't forsee us ever running Homu on a non-Unix platform, and because this is also hardcoded in the `homu/init.sls` file, let's just hardcode the whole path and inline it where it gets used.